config/deploy.php

```diff
@@ -101,7 +101,7 @@
         'application' => env('APP_NAME', 'Laravel'),
         'repository' => 'https://github.com/SlovakNationalGallery/webumenia.sk.git',
         'http_user' => 'www-data',
-        'php_fpm_service' => 'php7.2-fpm',
+        'php_fpm_service' => 'php7.4-fpm',
```

Comment:
[boulder]
```suggestion
        'php_fpm_service' => 'php7.4-fpm',
        'bin/php' => 'php7.4',
        'bin/composer' => 'php7.4 $(which composer)',
```
we needed to add this also here .. to keep deployer working properly -> https://github.com/SlovakNationalGallery/komarch.sk/pull/24/commits/b7c339ad998ef0d02840d20451513e5ee93b06b8

Reply:
Good catch!